### PR TITLE
Implement Matter command over MQTT (outgoing data)

### PR DESCRIPTION
**Pull request by MonicaisHer**
endpoint3:
```
$ sudo SERVER_ADDRESS="tcp://localhost:1883" TOPIC_PREFIX="matter-bridge-version0.1" out/debug/chip-bridge-app
$ sudo chip-tool onoff toggle 110 3
$ mosquitto_sub -h localhost -t "#" -v
# <topic-prefix>/<endpointID>/<clusterName>/<attributeName> {<payload>}
matter-bridge-version0.1/3/OnOff/OnOff {
        "attributeId" : 0,
        "clusterId" : 6,
        "command" : "on",
        "deviceName" : "Light 1",
        "endpointId" : 3,
        "location" : "Office",
        "parentEndpointId" : 1,
        "zone" : ""
}
```

endpoint2:
```
$ sudo chip-tool onoff toggle 110 2
CHIP:TOO: Run command failure: IM Error 0x0000057F: General error: 0x7f (UNSUPPORTED_ENDPOINT)
```
endpoint4:
```
$ sudo chip-tool onoff toggle 110 2
CHIP:TOO: Run command failure: IM Error 0x000005C3: General error: 0xc3 (UNSUPPORTED_CLUSTER)
```
endpoint5:
```
$ sudo chip-tool onoff toggle 110 5
CHIP:TOO: Run command failure: IM Error 0x000005C3: General error: 0xc3 (UNSUPPORTED_CLUSTER)
```

### CODE REVIEW

File: examples/bridge-app/linux/main.cpp [hunk 1]
```diff
@@ -552,19 +556,29 @@ EmberAfStatus HandleWriteOnOffAttribute(DeviceOnOff * dev, chip::AttributeId att
...
+        const std::string topic = std::to_string(endpointId) + "/" + clusterName + "/" + attributeName;
+
+        json payload;
+        payload["deviceName"] = deviceName; 
 
         if (*buffer)
         {
             dev->SetOnOff(true);
-            dev->MQTTPublish(*clientPtr, std::string(dev->GetName()) + "/" + topic, "ON");
+            payload["Command"] = "ON";
```
Comment (farshidtz) [inline examples/bridge-app/linux/main.cpp]:
```suggestion
            payload["command"] = "on";
```

File: examples/bridge-app/linux/main.cpp [hunk 2]
```diff
@@ -552,19 +558,43 @@ EmberAfStatus HandleWriteOnOffAttribute(DeviceOnOff * dev, chip::AttributeId att
...
+        char *envTOPIC_PREFIX = std::getenv(TOPIC_PREFIX);
+        if (envTOPIC_PREFIX == nullptr || strlen(envTOPIC_PREFIX) == 0)
         {
-            dev->SetOnOff(true);
-            dev->MQTTPublish(*clientPtr, std::string(dev->GetName()) + "/" + topic, "ON");
+            topicPrefix = "matter-bridge";
+            ChipLogProgress(DeviceLayer, "[MQTT] Using default TOPIC_PREFIX: %s", topicPrefix);
+        } else {
+            topicPrefix = envTOPIC_PREFIX;
+            ChipLogProgress(DeviceLayer, "[MQTT] Using TOPIC_PREFIX from environment variable: %s", topicPrefix);
```
Comment (farshidtz) [inline examples/bridge-app/linux/main.cpp]: Could be reduced to:
```
topicPrefix = (envTOPIC_PREFIX == nullptr || strlen(envTOPIC_PREFIX) == 0) ? "matter-bridge" : envTOPIC_PREFIX;
ChipLogProgress(DeviceLayer, "[MQTT] Using TOPIC_PREFIX: %s", topicPrefix);
```

File: examples/bridge-app/linux/include/nlohmann/json.hpp
Comment (farshidtz) [inline examples/bridge-app/linux/include/nlohmann/json.hpp:1]: Okay for development, but the source file of the library shouldn't be checked out here.

You may instead look into using jsoncpp, used in other examples and already part of the third party projects.
https://github.com/project-chip/connectedhomeip/blob/7ae5919a046a18c7e60537aff814977ee424b9cf/examples/all-clusters-app/linux/BUILD.gn#L41
https://github.com/project-chip/connectedhomeip/tree/master/third_party/jsoncpp

**Review by farshidtz — APPROVED** [review]

File: examples/bridge-app/linux/include/nlohmann/json.hpp
Comment (MonicaisHer) [reply examples/bridge-app/linux/include/nlohmann/json.hpp:1]: Do you suggest switching jsoncpp library from https://github.com/nlohmann/json to https://github.com/open-source-parsers/jsoncpp?

**Review by farshidtz — APPROVED** [review]
Looks good

**Pull request merged**